website/pages/ar/deploying/deploy-using-subgraph-studio.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Deploy Using Subgraph Studio
+---
+
+Learn how to deploy your subgraph to Subgraph Studio.
+
+> Note: When you deploy a subgraph, you push it to Subgraph Studio, where you'll be able to test it. It's important to remember that deploying is not the same as publishing. When you publish a subgraph, you're publishing it on-chain.
+
+## Subgraph Studio Overview
+
+In [Subgraph Studio](https://thegraph.com/studio/), you can do the following:
+
+- View a list of subgraphs you've created
+- Manage, view details, and visualize the status of a specific subgraph
+- Create and manage your API keys for specific subgraphs
+- Restrict your API keys to specific domains and allow only certain Indexers to query with them
+- Create your subgraph through the Studio UI
+- Deploy your subgraph using the The Graph CLI
+- Test your subgraph in the playground environment
+- Integrate your subgraph in staging using the development query URL
+- Publish your subgraph with the Studio UI
+- Manage your billing
+
+## Install The Graph CLI
+
+Before deploying, you must install The Graph CLI.
+
+You must have [Node.js](https://nodejs.org/) and a package manager of your choice (`npm`, `yarn` or `pnpm`) installed to use The Graph CLI. Check for the [most recent](https://github.com/graphprotocol/graph-tooling/releases?q=%40graphprotocol%2Fgraph-cli&expanded=true) CLI version.
+
+**Install with yarn:**
```

Comment:
Should these be headings? (Maybe it's fine as is if they look bad as headings, but h3s should be pretty small I think)